setup.sh

```diff
@@ -53,7 +53,7 @@ _os() {
 function _preparation() {
   echo "Updating system and grabbing core dependencies."
   if [[ $distribution = "Ubuntu" ]]; then
-    echo "Checking enabled repos"
+    echo "Fetching updates from enabled repos"
```

Comment:
Please don't change this. This is literally a check for enabled repos and to enable universe/multiverse if they are not already enabled